client/app/pages/users/index.js

```diff
@@ -12,6 +12,7 @@ export default function init() {
       path: '/users/new',
       title: 'Users',
       key: 'active',
+      isNewUserPage: true,
```

Comment:
Perhaps `openCreateUserDialog: true` will be more accurate on what this will do

Reply:
Yeah, I also called it something similar, but that name looks weird in context of line 157 (handle modal close). Let it stay `isNewUserPage` for now (if you don't mind of course).

Reply:
That'll do it for now